src/Tests/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs

```diff
@@ -61,6 +61,8 @@ public TestProject([CallerMemberName] string name = null)
 
         public Dictionary<string, string> AdditionalItems { get; } = new Dictionary<string, string>();
 
+        public List<TestItemElement> AdditionalItemElements { get; } = new();
```

Comment:
What is "ItemElements"? You don't need a separate method for doing it

Reply:
Is there a recommended pattern to do this? dotnet-watch has tests that adds items to a project with additional metadata:

```C#
<Compile Include="Blah" Exclude="Baz" Watch="false" />
```

Reply:
add more detail to my previous comment. You can use XElement.Parse and just put text in it